### ISSUE TITLE
MailTrackerBlocker breaks some links in emails

### ISSUE THREAD

**Issue by MrMage** [apparition47/MailTrackerBlocker#111]
It seems like this plugin removes parts of the domain in some emails. This might be to prevent click tracking, but I'd rather have click-tracked links than broken ones.

Example URL _before_ MTB has processed the email:

```
https://click.pstmrk.it/2m/my.paddle.com%2Freceipt<some other stuff>
```

Example URL _after_ MTB has processed the email:

```
https://click./2m/my.paddle.com%2Freceipt<some other stuff>
```

**Comment by apparition47** [comment]
Which version of MTB are you using? This should've been fixed in the latest 0.3.29

**Comment by MrMage** [comment]
Sorry, I should have checked that first. Works after updating to 0.3.29 👍
